brouter-core/src/main/java/btools/router/RoutingContext.java

```diff
@@ -226,6 +227,7 @@ public static void prepareNogoPoints( List<OsmNodeNamed> nogos )
         try { ir = Integer.parseInt( s.substring( 4 ) ); }
         catch( Exception e ) { /* ignore */ }
       }
+      // TODO[Phyks]
       nogo.radius = ir / 110984.; //  6378000. / 57.3;

```

Comment:
Is it expected to be Earth radius over (180 / PI) ? The value of the constant does not match with the comment :/

Reply:
Originally yes, but then I tuned to better distance results of other gpx tools

Reply:
Ok, thanks for the infos!